source/_lovelace/guage.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: "Gauge Card"
+sidebar_label: Gauge
+description: "The Gauge card allows you to display sensor information visually"
+date: 2018-10-11 10:28 +00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+Gauge card is a basic card that allows visually seeing sensor data
+
+<p class='img'>
+<img src='/images/lovelace/lovelace_gauge_card.gif' alt='Screenshot of the gauge card'>
+Screenshot of the gauge card.
+</p>
+
+{% configuration %}
+type:
+  required: true
+  description: gauge
+  type: string
+entity:
+  required: true
+  description: "Entity id to show"
+  type: string
+title:
+  required: false
+  description: Title of Gauge Data
+  type: string
+unit_of_measurement:
+  required: false
+  description: Unit of Measurement given to data
+  type: string
+  default: "Unit Of Measurement given by entity"
+min:
+  required: false
+  description: Minimum value for graph
+  type: number
+  default: 0
+max:
+  required: false
+  description: Maximum value for graph
+  type: number
+  default: 100
+severity:
+  required: false
+  description: Allows setting of colors for different numbers
+  type: object
+{% endconfiguration %}
+
+## {% linkable_title Severity Variables %}
+
+If you define the severity object, these are the required options:
+
+{% configuration %}
+green:
+  required: true
+  description: Value from which to start green color
+  type: number
+yellow:
+  required: true
+  description: Value from which to start yellow color
+  type: number
+red:
+  required: true
+  description: Value from which to start red color
+  type: number
+{% endconfiguration %}
+
+## {% linkable_title Examples %}
+
+Basic example:
```

Comment:
The basic example should be placed above the configuration variables list and should only contain the required parameters.